composer.json

```diff
@@ -20,7 +20,9 @@
         "cakephp/cakephp": "~3.0"
```

Comment:
CakePHP version needs to be bumped upto `^3.4`. For e.g. you have used `InstanceConfigTrait::setConfig()` in `PaginatorHelper` which isn't available in older versions.